parts/kuberneteskubelet.service

```diff
@@ -45,7 +45,8 @@ ExecStart=/usr/bin/docker run \
         --azure-container-registry-config=/etc/kubernetes/azure.json \
         --hairpin-mode=promiscuous-bridge \
         --network-plugin=${KUBELET_NETWORK_PLUGIN} \
-        --v=2
+        --feature-gates=Accelerators=true \
```

Comment:
@JackQuincy @seanknox Jack is right, stupid mistake on my part. The flag is passed from agentcustomdata, I'll fix that in a few moments.